tests/io/vasp/test_inputs.py

```diff
@@ -638,6 +639,33 @@ def test_as_dict_and_from_dict(self):
         incar3 = Incar.from_dict(dct)
         assert incar3["MAGMOM"] == [Magmom([1, 2, 3])]
 
+    def test_from_file_and_from_dict(self):
```

Comment:
I didn't see much documentation, wondering should init directly from a key: value dict be allowed at all (in contrast to the MSONable dict)?